apps/web/scripts/kernel/handlers/kernel.ts

```diff
@@ -29,5 +39,16 @@ export const onKernelMessage: MessengerCallback = async (payload, channel) => {
 				message: 'Invalid request type!',
 			});
 		}
+
+		const network = getNetwork(type);
```

Comment:
`network` should not a "required" piece, prefer clarify it as an parameter (if needed) instead of string naming convention.